### ISSUE TITLE
[Question] Get Total Count of Files within a Bucket ?

### ISSUE THREAD

**Issue by jeffward01** [coryrwest/B2.NET#51]
Hello!

Thank you for this great library! It is epic, I am writing a bit of software for the B2 team, and will use this library for it.

I see on my B2 Dashboard, I can view the total count of all files within a bucket:
![image](https://user-images.githubusercontent.com/12589359/155822868-27f38cbb-09ca-4d46-b4e0-994ad8f258c1.png)

Reviewing the B2 CLI API, I see we can view the total size of a Bucket:
[See here](https://github.com/Backblaze/B2_Command_Line_Tool#:~:text=b2%20get%2Dbucket%20%5B%2Dh%5D%20%5B%2D%2DshowSize%5D%20bucketName)

Reviewing the B2 Http API, I do not see an option to get the count of all files within a bucket:
[See here](https://www.backblaze.com/b2/docs/b2_list_buckets.html)

---

Is it possible to get the count of all files within a bucket?

Basically I want to add a progress bar, as retrieving files can take a long time, even using batches of 1000.

Thanks again!
